ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsService.scala

```diff
@@ -216,7 +216,10 @@ class ContractsService(
             jwt,
             ledgerId,
             parties,
-            resolvedTemplateIds,
+            // TODO Ray, maybe can ignore, and assume resolvedTemplateIds is template ids only
```

Comment:
Only if `SearchContext.ById` guarantees a `ContractTypeId.Template`.

Reply:
Not for this issue, but certainly for #14818, it ought to work with interface ID. So we can "assume resolvedTemplateIds is template ids only" here but it must be fixed for #14818.